ts/Core/Chart/ChartDefaults.ts

```diff
@@ -965,6 +965,55 @@ const ChartDefaults: ChartOptions = {
      */
     zoomBySingleTouch: false,
 
+    /**
+     * Chart zooming options.
```

Comment:
This needs `@since` and all options below should not define `@since`. It's not possible to declare:
```js
zooming: {
  singleTouch: true
}
```
In version 9.0.0, right? :)